docx/blkcntnr.py

```diff
@@ -52,12 +50,26 @@ def add_table(self, rows, cols, width):
         self._element._insert_tbl(tbl)
         return Table(tbl, self)
 
+    def add_sdt(self, tag_name):
```

Comment:
lets extend this one to receive alias name as well because when we generate content controls, alias name will be the one displayed to the user. let's make it optional, if the user specifies one, then it should be used, if not, it should default to the `tag_name` value

Reply:
Nice touch, it does look better now